.github/workflows/mypy.yml

```diff
@@ -13,14 +13,16 @@ jobs:
       - name: Setup APT
         run: |
           apt-get update
-          apt-get install --no-install-recommends -y ca-certificates
+          apt-get install --no-install-recommends -y ca-certificates gnupg
```

Comment:
Do we need gnupg here?

Reply:
Don't we need it to install the GPG key of the deb-ci repo?

Reply:
no, that's done by apt itself